CONTRIBUTING.md

```diff
@@ -34,6 +34,8 @@ You should now be able to run the unit tests in the `test` directory:
 python3 -m unittest
 ```
 
+> **NOTE:** files required to run the tests are in a git submodule `test/SBOLTestSuite`. These get downloaded if you clone the repo with `--recurse-submodules https://github.com/SynBioDex/pySBOL2.git`. If you have not, you can download them with `git submodule update --init` from the project root directory.
```

Comment:
```suggestion
**NOTE:** files required to run the tests are in a git submodule `test/SBOLTestSuite`. These get downloaded if you clone the repo with `--recurse-submodules https://github.com/SynBioDex/pySBOL2.git`. If you have not, you can download them with `git submodule update --init` from the project root directory.
```

I'd suggest without the quote bar. Otherwise, that's fine.